drivers/scheduler/anthos/anthos.go

```diff
@@ -200,6 +201,12 @@ func (anth *anthos) Init(schedOpts scheduler.InitOptions) error {
 	if err := anth.getVersion(); err != nil {
 		return err
 	}
+	if len(schedOpts.UpgradeHops) > 0 {
```

Comment:
nit: you can also do this instead
```suggestion
	if len(schedOpts.UpgradeHops) > 0 && if len(strings.Split(schedOpts.UpgradeHops, ",")) > 1 {
		anth.adminClusterUpgrade = true
	}
```